packages/perseus-core/src/parse-perseus-json/perseus-parsers/measurer-widget.ts

```diff
@@ -32,7 +32,5 @@ export const parseMeasurerWidget: Parser<MeasurerWidget> = parseWidget(
         rulerPixels: number,
         rulerLength: number,
         box: pair(number, number),
-        // TODO(benchristel): static is not used. Remove it?
-        static: defaulted(boolean, () => false),
```

Comment:
Nice!  My guess is that this snuck into the object data at some point by accident.